pkg/stomp/StompConsumer.php

```diff
@@ -96,17 +97,23 @@ public function receive(int $timeout = 0): ?Message
     {
         $this->subscribe();
 
-        if (0 === $timeout) {
-            while (true) {
-                if ($message = $this->stomp->readMessageFrame($this->subscriptionId, 100)) {
-                    return $this->convertMessage($message);
+        try {
+            if (0 === $timeout) {
+                while (true) {
+                    if ($message = $this->stomp->readMessageFrame($this->subscriptionId, 100)) {
+                        return $this->convertMessage($message);
+                    }
                 }
             }
-        } else {
-            if ($message = $this->stomp->readMessageFrame($this->subscriptionId, $timeout)) {
-                return $this->convertMessage($message);
+            else {
+                if ($message = $this->stomp->readMessageFrame($this->subscriptionId, $timeout)) {
+                    return $this->convertMessage($message);
+                }
             }
         }
+        catch (ErrorFrameException $e) {
+            throw new \Exception($e->getMessage() . "\n" . $e->getFrame()->getBody());
```

Comment:
The library must throw only exceptions that implement [interop exception interface](https://github.com/queue-interop/queue-interop/blob/master/src/Exception/Exception.php).